### ISSUE TITLE
MotoG6Play Bootloop AOSP 10.0 v219 (Magisk)

### ISSUE THREAD

**Issue by MarcosPearce** [phhusson/treble_experimentations#1350]
Moto g6 play is giving bootloop when installing magisk, but this error only happens from the version AOSP 10.0 v217.It doesn't leave the initial boot screen, so I can't get the logcat

**Comment by phhusson** [comment]
Yeah that's a known bug of magisk: it doesn't use system's secilc.
I can't really see how to work around it

Jun 6, 2020 7:29:06 PM MarcosPearce <notifications@github.com>:

**Comment by MarcosPearce** [comment]
Is there any way to use the modules without installing magisk? because if it exists I will abandon magisk, at least for now

**Comment by alt1478** [comment]
Nope

**Comment by alt1478** [comment]
Magisk modules need magisk

**Comment by MarcosPearce** [comment]
:(

**Comment by MarcosPearce** [comment]
I managed to get some logs I hope this helps in something

[console-ramoops.log](https://github.com/phhusson/treble_experimentations/files/4822315/console-ramoops.log)
[kmsg.log](https://github.com/phhusson/treble_experimentations/files/4822316/kmsg.log)
